pkg/controller/common.go

```diff
@@ -90,10 +90,10 @@ const (
 	pod              operationRequester = "pod-controller"
 )
 
-type cleanUpMode int
+type attachmentCleanUpMode int
```

Comment:
Should we rename these in this PR instead?

Reply:
Sure